### ISSUE TITLE
Remove operation is not reflected

### ISSUE THREAD

**Issue by alshakero** [Palindrom/palindrom-client#47]
All details here: https://github.com/Starcounter/Home/issues/250

**Comment by alshakero** [comment]
Bower is down for the meantime https://github.com/bower/bower/issues/1385. Will check it later.

**Comment by alshakero** [comment]
Closing as fixed in https://github.com/Palindrom/palindrom-polymer-client/pull/48

**Comment by alshakero** [comment]
This bug is back in Polymer 2.0, and is worse :( 

Am I doing anything wrong? https://jsbin.com/yefuxil/1/edit?html,output

@warpech @tomalec

**Comment by warpech** [comment]
I tried to follow the recommendation from https://github.com/Polymer/polymer/issues/4652 and it still does not work: https://jsbin.com/xivutoxoku/1/edit?html,output

https://github.com/Polymer/polymer/issues/3892 mentions "overriding the dirty checking as per the documentation" but I can't find the meaning of this. Maybe they mean [this](https://github.com/Polymer/polymer/blob/master/lib/mixins/property-effects.html#L1324-L1353)

@alshakero can you escalate this to the Polymer issue?

**Comment by alshakero** [comment]
> overriding the dirty checking as per the documentation

Most probably they mean this https://www.polymer-project.org/1.0/docs/devguide/model-data#override-dirty-check.

**Comment by tomalec** [comment]
IT seems it may not bee easy with Polymer https://github.com/Polymer/polymer/pull/4999#issuecomment-357517032

**Comment by tomalec** [comment]
@alshakero is it still ⏰ ?

**Comment by warpech** [comment]
I understand this error still exists, but I don't recall a consequence in any Starcounter app because of it. I think it is no longer a ⏰. @alshakero pls confirm.

**Comment by alshakero** [comment]
I used ⏰ because when we were using Polymer 1.x this issue affected a real app and I had to fix it with a higher priority. So I naturally assumed to bring it back would be a big deal. Apparently it's not. 

We can close this but I'm pretty sure it will resurface soon. Especially when blending becomes prominent. Because we'll removing views (full objects, not primitives) from the model-view a lot. And they'll stick in the UI.
